### ISSUE TITLE
CLR Fails to Cleanup

### ISSUE THREAD

**Issue by skylerknecht** [Tw1sm/PySQLRecon#1]
https://github.com/Tw1sm/PySQLRecon/blob/498d05283fcb7ef6412f1128d2f7f38760fdf492/pysqlrecon/modules/clr.py#L116C2-L116C2

When exeucting the assesmbly it'll hit an exception if the assembly already exists. 

```
[21:57:44] ERROR    (SQL01): Line 1: CREATE ASSEMBLY failed because the source assembly is, according to MVID, identical to an assembly that is already registered under the name "nyEfEBML".                                                                                             
```

This execption needs to be caught, logged followed by the assembly being removed.

**Comment by Tw1sm** [comment]
I'll look into this. At the least, I should be able to add a `deleteclr` command so that the existing assembly can be easily removed

**Comment by Tw1sm** [comment]
Fixed in 21aad833824b9778c43554c4ef1baa917aae3426

Line that dies with that error was [here](https://github.com/Tw1sm/PySQLRecon/blob/498d05283fcb7ef6412f1128d2f7f38760fdf492/pysqlrecon/modules/clr.py#L89):
```
#####
# Create the custom assembly
pysqlrecon.create_asm(asm_name, dll_bytes)
```

Added better handing so that the offending custom assembly is deleted and a second attempt is made to create it. Example:
```
[22:03:29] INFO     Connecting to 192.168.0.235:1433
[22:03:29] INFO     Performing CLR custom assembly attack on 192.168.0.235
[22:03:29] INFO     Added SHA-512 hash for 'CreateProcess.dll' to sys.trusted_assemblies with a random name of 'cnmTDGPQ'
[22:03:29] INFO     Creating a new custom assembly with the name 'klttClIB'
[22:03:29] ERROR    (TRETOGOR): Line 1: CREATE ASSEMBLY failed because the source assembly is, according to MVID, identical
                    to an assembly that is already registered under the name "ZnKXxUAN".
[22:03:29] WARNING  Duplicate assembly detected - will try to delete and re-create
[22:03:29] INFO     Deleted the offending duplicate assembly 'ZnKXxUAN'
[22:03:29] INFO     Attempting to re-create assembly with name 'klttClIB'
[22:03:29] INFO     Loading DLL into stored procedure 'CreateProcess'
[22:03:29] INFO     Created '[klttClIB].[StoredProcedures].[CreateProcess]'
[22:03:29] INFO     Executing payload...
[22:03:29] INFO     Cleaning up...
```

**Comment by Tw1sm** [comment]
Please reopen if problem persists 👍
